src/frontend/src/lib/components/convert/ConvertInputsContainer.svelte

```diff
@@ -0,0 +1,18 @@
+<div class="flex flex-col items-start justify-between md:flex-row">
+	<!-- TODO: Update colors with values from theme-variables when they are available there -->
+	<div
+		class="mb-3 flex h-14 w-full items-center rounded-lg border-[1px] border-bright-gray bg-ghost-white px-3 md:mb-0 md:w-[35%]"
```

Comment:
i think `border-[1px]` is just `border`

Reply:
plus, if 33.3% is ok too, you can use directly `w-4/12`

Reply:
thanks!